### ISSUE TITLE
Missing @JsonProperty in Java model classes

### ISSUE THREAD

**Issue by afrolovsky** [swagger-api/swagger-codegen#6911]
##### Description

Generated Java model classes miss ```@JsonProperty``` annotation

##### Swagger-codegen version

Latest build from master branch (92a1c3a4)

##### Swagger declaration file content or url
https://gist.github.com/afrolovsky/039dcde61decbd092a0948833eefa6c0

##### Command line used for generation
```
@java -jar swagger-codegen-cli.jar generate ^
  --input-spec "swagger.json" ^
  --lang jaxrs-spec ^
  --output temp
```

##### Steps to reproduce

1. Save gist in a file named ```swagger.json```
2. Create ```run.cmd``` with the code from the previous section
3. Run it
4. Check ```temp\src\gen\java\io\swagger\model\TokenResponse.java```

When the TokenResponse object becomes a text, it has an "accessToken" property but not "access_token".

##### Related issues/PRs

https://github.com/swagger-api/swagger-codegen/issues/3612

##### Suggest a fix/enhancement
-

**Comment by wing328** [comment]
Would you have time to contribute a fix? Here is a good starting point: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/JavaJaxRS/spec/pojo.mustache

pojo.mustache for other Java server generator might have the issue fixed already so I would also suggest you to take a look at those.

**Comment by wing328** [comment]
Fixed via #6916
